### PR TITLE
[anti-whale] Add anti-whale strategy

### PR DESCRIPTION
**Pull request by joaomajesus**
Changes proposed in this pull request:
- add an anti-whale strategy that executes a configured strategy and applies an anti-whale measure to its results to reduce the impact of big wallets in the the resulting value, reducing the effect on the voting power as the token amount increases.

### CODE REVIEW

File: src/strategies/anti-whale/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    result = result * thresholdMultiplier
+  }
+  ```
+
+## Accepted options
+
+- **strategy.name:** name of the strategy to run
+
+- **strategy.network:** the network id to be used by the strategy.
```
Comment (bonustrack) [inline src/strategies/anti-whale/README.md:24]: The issue with that is that the space has already a network defined, and the snapshot block number is specific to this network, if you use Polygon network on the strategy but the space use mainnet the snapshot will be completely off. I'd rather not add any network param here.

Comment (joaomajesus) [reply src/strategies/anti-whale/README.md:24]: ahh I see.... good one, I guess that's why the multichain strategy made some calls to the Graph to get the blocks for each network :)

Comment (bonustrack) [reply src/strategies/anti-whale/README.md:24]: Exactly!

Comment (joaomajesus) [reply src/strategies/anti-whale/README.md]: It's done. Thanks for the help!

File: src/strategies/anti-whale/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import getProvider from '../../utils/provider';
```
Comment (ChaituVR) [inline src/strategies/anti-whale/index.ts:1]:
```suggestion
```

**Review by ChaituVR — COMMENTED** [review]
LGTM

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Thanks for the PR :) strategy is live now https://snapshot.org/#/strategy/anti-whale

**Pull request merged**